.github/workflows/km-ci-workflow.yaml

```diff
@@ -56,7 +56,7 @@ on:
         required: true
 
 env:
-  BUILDENV_IMAGE_VERSION: latest # use this for all buildenv containers
+  BUILDENV_IMAGE_VERSION: PR_1436 # use this for all buildenv containers
```

Comment:
Is this for debugging?

Reply:
This is to make this CI run use the new buildenv while maintaining the old one for the rest of the runs. It is documented in `image-targets.md`